### ISSUE TITLE
toki

### ISSUE THREAD

**Issue by Th3Scribble** [lipu-linku/pali-nimi#16]
(copying from the original toki post)

main ideas:
- toki is communication
- toki doesn't require an external recipient, so there should be examples that imply one and examples that don't

proposed definition:
> communicate, speak, tell, ponder, discuss; language

the semicolon is to show that while all the definitions before it are forms of "communicate", "language" is an extension of that to "form of communication"
i didn't include any noun examples but i'm not sure what the best translation for that is, since i don't really have a word for that in english. where i would say "toki sina li pona" i would say "what you said is good", not something like "your speech is good". would "what is said" or "what one says" be a good inclusion?

---

#### sona pu

VERB to communicate, say, speak, say, talk, use language, think

#### sona Linku pi toki Inli

communicate, say, speak, talk, use language, think; hello

#### sona Linku pi toki pona

toki la, jan li ken pana e sona kepeken kalama anu sitelen anu luka.

#### sona ku

communicate⁵, speak⁵, say⁵, conversation⁵, communication⁵, hello⁵, speech⁵, talk⁵, language⁵, tell⁵, statement⁵, hi⁵, discuss⁵, discussion⁵, discourse⁴, mention⁴, comment⁴, refer⁴, message⁴, express³, remark³, dialogue³, reporting³, call³, narrative³, contact³, respond³, commentary³, disclose³, proposal³, admit³, verbal³, tale³, story³, declare³, interview³, testify³, answer³, report³, expression³, phrase³, excuse², hey², response², reply², greet², confess², declaration², interaction², suggest², quote², state², preach², describe², negotiate², specify², claim², feedback², counsel², coverage², proposition², suggestion², accuse², consult²

**Comment by gtbot2007** [comment]
conversation for a noun?

**Comment by gregdan3** [comment]
personally I feel it's important that "think" in particular is included in the definition

something that was brought up in the original issue is that we should have both examples of to-self "communication" and not-to-self communication

the last proposed definition of toki was:

> tell, speak, story, think, inform, speech, discuss, language, communicate, announcement, conversation

given how the rest of the word discussion has gone so far, and a bit of consideration for brevity and focus, i'm gonna propose:

> tell, speak, story, think, inform, discuss, language, announcement, conversation; (interjection) hello

which gives us a good variety of verbs/nouns 

i also added `(interjection) hello` which isn't necessary, and could be redone tons of ways. examples:

- (sentence word) hello
- (interjection) [greeting]
- (sentence word) [greeting]
- (interjection) hello, hi, good morning, ...

please iterate on this if y'all feel like this should be included! i'm unsure which is most clear but personally i lean toward the one i put in the def initially, or [greeting] after that

**Comment by lipamanka** [comment]
I agree about including think, definitely. I think because all words can be used as interjections including toki here contributes to an uneven and lexicalized approach to interjections, so I'd prefer it out, but it can stay in if someone feels strongly about it. it *is* in use and it *is* in pu.

**Comment by janPensa** [comment]
My first impression of "(interjection) hello" was leaning towards not including it, but on the other hand, knowing the greeting "toki!" is very useful to beginners, and that usage is not easily derived from the other meanings of the word.

But on the _other_ other hand, most beginners would probably learn it quick enough from courses and other beginner-oriented resources.

**Comment by janPensa** [comment]
as for

> tell, speak, story, think, inform, discuss, language, announcement, conversation

I feel like "inform" and "announcement" are slightly weak examples compared to the others. For some reason, I think "announce" is much more toki-like than "announcement", and "announce" is close to "inform", so we could merge those two into just "announce".

I also think "say" and "talk" are better, more common examples than "tell". And also far less likely to result in beginners making mistakes like "I told him" -> "mi toki e ona"

So I'd propose

> say, speak, talk, think, announce, discuss, language, story, conversation

(And possibly "(interjection) hello" or similar at the end, depending on what people want)

**Comment by gregdan3** [comment]
honestly, i'm very mixed on including the greeting, so here's my current breakdown:

### reasons to include
- toki greeting is in the current definition as "hello"
- this is a use that is as widespread as it gets
- it is an appropriate definition of the word
- we have defined words specifically in the interjection position before
- it's important for beginners to know
- the definition search will show this to people immediately (this might be the most important thing to me on this list)

## reasons to exclude
- this is arguably a lexicalization, although if it is analyzed that way it's the Oldest one to the point of having faded into the background
- there are other ways of saying hello, especially that have cropped up in the recent few months, and "hiding" the definition of hello from toki encourages these
- it makes the whole definition shorter, giving us more room to expand the core definition of the word (note that this is more "comfort" length than any strict length limit we're dealing with)

hmm i had felt like ther were more reasons to exclude but there were not that i could think of

---

@janPensa 
> For some reason, I think "announce" is much more toki-like than "announcement", and "announce" is close to "inform", so we could merge those two into just "announce".

for me, "announcement" is closest to "conversation", being a noun representing some discrete kind of talking. given that, i think they're all appropriate. in fact i already took out the much more boring "communicate" with similar thinking, given its similarity to all the others already there

**Comment by wasokeli** [comment]
i'd kinda like "communicate" to be in there, to be inclusive of 
- forms of nonverbal communication, like body language (sijelo li toki e lete)
- metaphorical uses, like a patch of grass growing greener over damp soil (kule kasi li toki e telo)

**Comment by wasokeli** [comment]
redundancies in my view:
say, speak
discuss, conversation
i like "say" because it takes a direct object in the same way as "toki", and i like "conversation" because it's a common noun usage.

i like jan Seli's suggestion (in the first post) to append "language" after a semicolon. either that or the pu system of saying "use language" as a verb.

**Comment by wasokeli** [comment]
i think "communicate" covers "inform" and "announce", too. so
> communicate, say, think, conversation, story; language

**Comment by lipamanka** [comment]
why are we putting the semicolon in the middle of our noun examples?

**Comment by lipamanka** [comment]
why are we putting the semicolon in the middle of our noun examples?

**Comment by wasokeli** [comment]
because every word here is an example of a communication except "language"

**Comment by lipamanka** [comment]
alright if that's what everyone else wants here

**Comment by lipamanka** [comment]
alright if that's what everyone else wants here

**Comment by gregdan3** [comment]
hmm, I don't get why language should be split with a semicolon

It's a different sense of the word toki, but we have been using semicolon to divide functions, not senses, right?

ko for example has no semicolons right now, because it covers adjectives and nouns that mesh into one idea; there's no transitive ko in that definition. this is in spite of the fact that while you can bundle everything in ko under "semi-solid" as we've done, to most English speakers ko comes across as centering around two fairly separate ideas: powders and goops.

as an even stronger example of this: I don't think any of us would agree that "laso" should be anything like "green, vermillion; blue, cyan"

so... why are we doing this to toki? or is there a reason this comparison doesn't apply?

**Comment by wasokeli** [comment]
toki Tepo addressing this from the previous thread:
> Really a language is a nasin toki and this is an example of the little known variant on tenpo dropping called nasin dropping la pilin mi la ona li wile lon poki ante.
https://github.com/lipu-linku/pali-nimi/issues/1#issuecomment-1674254833

**Comment by lipamanka** [comment]
also where did story come from? it might've come from me but if it did I forgot about that. do we need it? Also the nasin dropping thing is probably a joke, right? like that's not a thing that happens. right?

**Comment by wasokeli** [comment]
story is from toki Kekan ni: https://github.com/lipu-linku/pali-nimi/issues/1#issuecomment-1675065361

**Comment by wasokeli** [comment]
i like the inclusion of "story", it clarifies something i was confused about as a learner, but i'm okay with removing it

**Comment by wasokeli** [comment]
language la
i think it's reasonable to think of a language as a nasin of communication, an ilo of communication, or a ken of communication.
- other ilos of communication, like walkie-talkies, aren't called toki.
- other nasins of communication, like looking your conversation partner in the eyes, aren't called toki.
- other kens of communication, like walking closer so you can hear each other, aren't called toki.

**Comment by lipamanka** [comment]
i would call conveying information through eye contact a type of toki personally. even though "nasin" means "method," toki can also mean "method of communication" as well as "instance of communication." but i do see how this makes the language aspect of toki a bit more separate, i just don't like the semicolon here. 

re: story, 👍 thanks for the explain
